### DISCUSSION TITLE
Welcome to realm-js Discussions!

### DISCUSSION THREAD

**Discussion by geragray** [realm/realm-js#3459] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Ngomana** [comment]
Is it possible to export default configuration file and use it other arrears like so in typescript?
```
import Realm from "realm";
import { getRealmApp } from "../functions/realmConfig";
import { ItemSchema } from "./itemSchema";

export const app = getRealmApp();
export const useRealmApp = getRealmApp();

export const user = useRealmApp.currentUser;
export const partitionValue = useRealmApp.currentUser.id;

export const config = {
  schema: [ItemSchema], //other schema will be added in the future
  sync: {
    //@ts-ignore
    //@ts-nocheck
    user: user,
    partitionValue: partitionValue, //app.currentUser.id,
  },
};

export const useRealm = Realm.open(config);

export const testRealm = () => {
  if (user === null) {
    console.log("fuck realm");
    return;
  }

  //@ts-ignore
  return useRealm;
};
```
